src/components/InRowAction/InRowAction.js

```diff
@@ -68,6 +69,15 @@ const InRowAction = props => {
             <i className="fas fa-edit"></i>
           </button>
         ) : null}
+

```

Comment:
Why add this? What's the difference between deleteCompetencyModel and deleteModal? editCompetencyModel and editModal?

Reply:
Apparently, I have not used deleteModal and editModal, which lead to creating another prop/s in InRowAction.